samples/Makefile

```diff
@@ -11,40 +11,74 @@ generate:
 	@echo  $(TEST_PKGS) $(TEST_DB_PKG)
 	
 	../bin/wag -with-tests -file ./swagger.yml -output-path ./gen-go-basic -js-path ./gen-js
-	cd ./gen-js && jsdoc2md index.js types.js > ./README.md
+	cd ./gen-js && jsdoc2md index.js types.js > ./README.md    
+	cd ./gen-go-basic/client && go mod tidy
+	cd ./gen-go-basic/models && go mod tidy
```

Comment:
We need to properly fill out submodule dependencies and we weren't doing that which was one cause of these tests being unable to run because client/models were not compilable.